stdlib/public/Platform/tgmath.swift.gyb

```diff
@@ -102,11 +132,27 @@ func _vjpExp(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
   return (value, { v in value * v })
 }
 
+@usableFromInline
+func _vjpExp2(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  let value = exp2(x)
+  return (value, { v in v * ${T}(M_LN2) * value })
+}
+
 @usableFromInline
 func _vjpLog(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
   return (log(x), { v in v / x })
 }
 
+@usableFromInline
+func _vjpLog10(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (log10(x), { v in v * ${T}(M_LOG10E) / x})
+}
+
+@usableFromInline
+func _vjpLog2(_ x: ${T}) -> (${T}, (${T}) -> ${T}) {
+  return (log2(x), { v in v / (${T}(M_LN2) * x)})
```

Comment:
```suggestion
  return (log2(x), { v in v / (${T}(M_LN2) * x) })
```

Reply:
done in 6b1ebf264b6801e5becf1b9f5d6206ce627af1f6